saltgui/static/stylesheets/page.css

```diff
@@ -101,6 +100,14 @@ table tr td:last-of-type {
   padding-right: 0;
 }
 
+table tr td.starttime,
+table tr td.target {
+  white-space: pre-wrap;
+  max-width: 20%;
+  width: 20%;
+  display: table-cell;
```

Comment:
Isn't it already a `table-cell`?

Reply:
Yes, it's not required. I will remove it.